src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -64,9 +64,12 @@ public PersistentDataStore() {
       try {
         UUID uuid = UUID.fromString((String) entity.getProperty("uuid"));
         String userName = (String) entity.getProperty("username");
+	String password = (String) entity.getProperty("password");
```

Comment:
Nit: Fix this indentation.

Reply:
I think I changed this in a new commit. The one spacing mistake wasn't showing for me when I went to change it, but I think it is all fixed now

Reply:
It looks like it's still on different indentation from the rest of the neighboring code.

Are you sure you changed it?